server/tests/auth-signin.test.js

```diff
@@ -0,0 +1,59 @@
+import chai, { expect } from 'chai';
+import { describe, it } from 'mocha';
+import chaiHttp from 'chai-http';
+import app from '../index';
+import 'dotenv';
+import { users } from '../config/fixtures.json';
+
+chai.should();
+chai.use(chaiHttp);
+
+
+describe('Test sign in', () => {
+  it('password is invalid', (done) => {
+    chai.request(app)
+      .post('/api/v1/signin')
+      .send(users[1])
+      .end((err, res) => {
+        expect(res).to.have.status(422);
+        done();
+      });
+  });
+  it('email not found', (done) => {
+    chai.request(app)
+      .post('/api/v1/signin')
+      .send(users[2])
+      .end((err, res) => {
+        expect(res).to.have.status(404);
+        done();
+      });
+  });
+  it('email is empty', (done) => {
+    chai.request(app)
+      .post('/api/v1/signin')
+      .send(users[3])
+      .end((err, res) => {
+        expect(res).to.have.status(422);
```

Comment:
You should test all response elements such as the returned message as well.

Reply:
Resolved and tested